backend/webapp/urls.py

```diff
@@ -6,7 +6,7 @@
 from django.urls import include, re_path
 from rest_framework.routers import DefaultRouter
 
-from catalogue.viewsets import ClassificationViewset, DescriptorViewset, BaseLayerViewset, LayerViewset, GroupViewset, GroupPriorityViewset, OrganisationViewset, BaseLayerGroupViewset
```

Comment:
Similar qualified import (import-as) could be used here.  (I know this is my fault that I'm asking you to clean up!)

Reply:
Imports cleaned-up